chalice/cli/__init__.py

```diff
@@ -283,13 +287,29 @@ def generate_sdk(ctx, sdk_type, outdir):
 
 
 @cli.command('package')
-@click.argument('outdir')
+@click.option('--single-file', is_flag=True,
+              default=False,
+              help=("Create a single packaged file. "
+                    "By default, the 'out' argument "
+                    "specifies a directory in which the "
+                    "package assets will be placed.  If "
+                    "this argument is specified, a single "
+                    "zip file will be created instead."))
```

Comment:
May want to include information on when to use it. Was not completely obvious on why it should be used.